scripts/aks/prepare_aks_artifacts.sh

```diff
@@ -78,7 +89,12 @@ else
   sed -i "s#DEPLOYMENT_ENVIRONMENT_PLACEHOLDER#integ#g" "${OUTPUT_TEMPLATE_FILE}"
   cat ${OUTPUT_TEMPLATE_FILE}
 
-  python3 ${ROOT}/add_env.py ${OUTPUT_TEMPLATE_FILE} uid2-operator CORE_BASE_URL http://$BORE_URL_CORE OPTOUT_BASE_URL http://$BORE_URL_OPTOUT SKIP_VALIDATIONS true
+  if [ "${TARGET_ENVIRONMENT}" == "mock" ]; then
+    python3 ${SHARED_ACTIONS_ROOT}/scripts/aks/add_env.py ${OUTPUT_TEMPLATE_FILE} uid2-operator CORE_BASE_URL http://$BORE_URL_CORE OPTOUT_BASE_URL http://$BORE_URL_OPTOUT SKIP_VALIDATIONS true
+  else
+    python3 ${SHARED_ACTIONS_ROOT}/scripts/aks/add_env.py ${OUTPUT_TEMPLATE_FILE} uid2-operator CORE_BASE_URL https://$BORE_URL_CORE OPTOUT_BASE_URL https://$BORE_URL_OPTOUT SKIP_VALIDATIONS true
```

Comment:
Could we do some templating? The difference is impossible to spot. Maybe use a `protocol` variable?